solr/core/src/java/org/apache/solr/handler/admin/CoreAdminHandler.java

```diff
@@ -233,7 +231,7 @@ protected void handleCustomAction(SolrQueryRequest req, SolrQueryResponse rsp) {
       .put(CoreAdminParams.SHARD, CoreDescriptor.CORE_SHARD)
       .put(CoreAdminParams.COLLECTION, CoreDescriptor.CORE_COLLECTION)
       .put(CoreAdminParams.ROLES, CoreDescriptor.CORE_ROLES)
-      .put(CoreAdminParams.CORE_NODE_NAME, CoreDescriptor.CORE_NODE_NAME)
+      .put(REPLICA_NAME, REPLICA_NAME)
```

Comment:
What's the purpose of having dupe param names for the other variables, like [SHARD](https://github.com/apache/lucene-solr/blob/master/solr/solrj/src/java/org/apache/solr/common/params/CoreAdminParams.java#L79)/[CORE_SHARD](https://github.com/apache/lucene-solr/blob/master/solr/core/src/java/org/apache/solr/core/CoreDescriptor.java#L58)?